controllers/userStatus.js

```diff
@@ -143,4 +145,52 @@ const updateAllUserStatus = async (req, res) => {
   }
 };
 
-module.exports = { deleteUserStatus, getUserStatus, getAllUserStatus, updateUserStatus, updateAllUserStatus };
+const cancelOOOStatus = async (req, res) => {
+  const userId = req.userData.id;
+  try {
+    const responseObject = await userStatusModel.cancelOooStatus(userId);
+    return res.status(200).json(responseObject);
+  } catch (error) {
+    logger.error(`Error while cancelling the ${userState.OOO} Status : ${error}`);
+    if (error instanceof Forbidden) {
+      return res.status(403).json({
+        statusCode: 403,
+        error: "Forbidden",
+        message: error.message,
+      });
+    } else if (error instanceof NotFound) {
+      return res.status(404).json({
+        statusCode: 404,
+        error: "NotFound",
+        message: error.message,
+      });
+    }
+    return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
```

Comment:
This line is not covered in test coverage

Reply:
This will happen when the call to Firestore DB fails. I am not sure how to simulate that 😢

Reply:
Done.